internal/receiver/smartagentreceiver/receiver.go

```diff
@@ -60,7 +60,7 @@ var (
 )
 
 func init() {
-	rusToZap = newLogrusToZap()
+	rusToZap = newLogrusToZap(nil)
```

Comment:
I'm still not sure this init block is necessary and that we shouldn't base our default logger on the first receiver instances.  We aren't complying w/ app log level otherwise.

I can take a crack at it in a later PR if you don't see a quick way/want to go for it.

Reply:
I wasn't aware the current approach is something we're looking to update. So, we would ideally use the first receiver logger's settings for any subsequent receivers?

Reply:
There was some back and forth in the initial PR, but the experience needs improvement imo.  thanks for taking care of this

Reply:
I played around a bit with this and I wasn't able to find a way to use the first receiver's settings to create the default logger as the receiver name field on the logger cannot be updated (relevant info: https://github.com/uber-go/zap/issues/765#issuecomment-558425090). However, like you suggested offline, I've set the level of the default logger based on the settings from the first receiver instance.